### ISSUE TITLE
Sylius 1.12.5 and Symfony 6: Web UI fails with Controller "sylius.controller.import_data:importFormAction" does neither exist as service nor as class

### ISSUE THREAD

**Issue by olimination** [FriendsOfSylius/SyliusImportExportPlugin#290]
If I try to open the "Products" or "Taxons" view in the admin ui, I receive the following error:

```
An exception has been thrown during the rendering of a template ("The controller for URI "/_fragment" is not callable: Controller "sylius.controller.import_data:importFormAction" does neither exist as service nor as class.").
```

I have enabled the `web_ui: true` config which produces this issue. Setting it to `false` the admin views render properly.

**Comment by olimination** [comment]
PR with fix is here: https://github.com/FriendsOfSylius/SyliusImportExportPlugin/pull/291

**Comment by pawellen** [comment]
Having same issue. Any updates on that?

**Comment by oallain** [comment]
Hello,

Just released [v0.24.0](https://github.com/FriendsOfSylius/SyliusImportExportPlugin/releases/tag/0.24.0) .

Regards
